gamekult.com.txt

```diff
@@ -1,15 +1,25 @@
-title: //article[@id='story-page']/h1
+title: //h1[@class="ed__news__title"] | //h4[@class='gk__helpers__fat-title-m--complementary ed__show-h__mdb__title']
 date: //p[@class="byline"]/time/@datetime
-body: //div[@class="gk__content__container"] | //div[@class="js-pm-refreshed-content"]
+body: //div[@class="gk__content__container"] | //div[@class="js-pm-refreshed-content"] | //article[contains(@class, "ed__review__article")]
 
 strip: //aside[@id="beginning"]
 strip: //select[@id="news_changing_node"]
+strip: //section[@class="gk__pt js-pt-container has--tbs is--md gk__hide"]
+strip: //aside[@class="ed__show__aside"]
+strip: //div[@class="gk__breadcrumb"]
+strip: //p[@class="ed__header__info--top"]
+strip: //div[contains(@class, "ed__review__article__header__price")]
+strip: //ul[contains(@class, 'js-preview-list')]
+replace_string(<footer>): <div>
+replace_string(</footer>): </div>
```

Comment:
What are these empty replacements?

Reply:
It's not empty replacement. If the `<footer>` element is present, I get the `padding-left` from the `material.css`.

Reply:
Sorry. Using the GitHub iOS app, it is rendered as empty 🤷
![image](https://user-images.githubusercontent.com/62333/78717127-69d98980-7920-11ea-92b7-d474fd8a9765.jpeg)

Reply:
Ah ah, no worries!